Objects/unicodeobject.c

```diff
@@ -11611,6 +11611,24 @@ unicode_index(PyObject *self, PyObject *args)
     return PyLong_FromSsize_t(result);
 }
 
+/*[clinic input]
+str.isascii as unicode_isascii
+
+Return True if the string is an ascii string, False otherwise.
```

Comment:
Style nit: You've used all uppercase version (ASCII) in the documentation and NEWS file. I'd suggest using the same style (whether ASCII or ascii) everywhere.